File: CS/13_Conversion/ToSVG.cs
using System;
using System.Windows.Forms;
using Spire.Pdf;

namespace ToSVG
{
    public partial class Form1 : Form
    {
        public Form1()
        {
            InitializeComponent();
        }

        private void button1_Click(object sender, EventArgs e)
        {
            //Pdf file
            String file = @"..\..\..\..\..\..\Data\ToSVG.pdf";

            //Open pdf document
            PdfDocument doc = new PdfDocument();
            doc.LoadFromFile(file);

            //Convert to svg file
            doc.SaveToFile("ToSVG-result.svg", FileFormat.SVG);
            doc.Close();
            PDFDocumentViewer("ToSVG-result.svg");
        }
        private void PDFDocumentViewer(string fileName)
        {
            try
            {
                System.Diagnostics.Process.Start(fileName);
            }
            catch { }
        }
    }
}

File: VB/13_Conversion/ToSVG.vb
Imports Spire.Pdf

Namespace ToSVG
	Partial Public Class Form1
		Inherits Form
		Public Sub New()
			InitializeComponent()
		End Sub

		Private Sub button1_Click(ByVal sender As Object, ByVal e As EventArgs) Handles button1.Click
			' Specify the file path of the PDF document to be loaded
			Dim file As String = "..\..\..\..\..\..\Data\ToSVG.pdf"

			' Create a new PdfDocument object
			Dim doc As New PdfDocument()

			' Load the PDF document from the specified file path
			doc.LoadFromFile(file)

			' Convert the PDF document to an SVG file format
			doc.SaveToFile("ToSVG-result.svg", FileFormat.SVG)

			' Close the PDF document
			doc.Close()

			' Launch the file
			PDFDocumentViewer("ToSVG-result.svg")
		End Sub
		Private Sub PDFDocumentViewer(ByVal fileName As String)
			Try
				Process.Start(fileName)
			Catch
			End Try
		End Sub
	End Class
End Namespace
